### PR TITLE
Self Installing MicroOS container host and k3s images

### PR DESCRIPTION
**Pull request by apinter**
Hi,

Long time user, first time contributor.  
The self installing images were available for a while, but doesn't seem to receive much love. Adding these images under `Alternative Downloads`.

### CODE REVIEW

**Comment by hellcp** [comment]
We should discuss adding a container section (#8) on the main page, and also container links/commands that you can easily copy on the pages of distributions (#4). Maybe we could also sync up on #3, since that could use a similar UI. I'm not that good at those things, so I would appreciate every bit of help

File: _includes/icons/selfinstalling.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...10-10zm0 2c0.44823 0 0.88435 0.046315 1.3125 0.11719l-0.65625 3.9375a4 4 0 0 0-0.65625-0.054688 4 4 0 0 0-0.060547 0 4 4 0 0 0-3.9395 4 4 4 0 0 0 0.67188 2.2188l-3.3262 2.2188c-0.84864-1.2701-1.3457-2.7955-1.3457-4.4375 0-4.418 3.582-8 8-8zm1.5605 0.1543c0.034031 0.0067371 0.067688 0.014317 0.10156 0.021484a8 8 0 0 0-0.10156-0.021484zm0.77344 0.19336c0.034744 0.010586 0.068975 0.022159 0.10352 0.033203a8 8 0 0 0-0.10352-0.033203zm0.19336 0.066406c0.68156 0.22707 1.324 0.53998 1.9102 0.93164l-2.2188 3.3262a4 4 0 0 0-0.95508-0.4668l1.2637-3.791zm4.127 3.1484c0.84864 1.2701 1.3457 2.7955 1.3457 4.4375 0 4.418-3.582 8-8 8-0.44823 0-0.88435-0.046315-1.3125-0.11719l0.65625-3.9375a4 4 0 0 0 0.65625 0.054688 4 4 0 0 0 4-4 4 4 0 0 0-0.67188-2.2188l3.3262-2.2188zm-6.7051 2.4375a2 2 0 0 1 0.001953 0 2 2 0 0 1 0.048828 0 2 2 0 0 1 2 2 2 2 0 0 1-2 2 2 2 0 0 1-2-2 2 2 0 0 1 1.9492-2zm-2.168 5.3281a4 4 0 0 0 0.95508 0.4668l-1.2637 3.791c-0.68155-0.22707-1.324-0.53998-1.9102-0.93164l2.2188-3.3262zm-0.21875 4.291a8 8 0 0 0 0.10352 0.033203c-0.034743-0.010586-0.068975-0.022159-0.10352-0.033203zm0.77539 0.20508a8 8 0 0 0 0.10156 0.021484c-0.034031-0.006737-0.067688-0.014317-0.10156-0.021484z"/></svg>
```
Comment (hellcp) [inline _includes/icons/selfinstalling.svg:1]: I would probably go with install icon from https://remixicon.com/#device. Remember to replace fill value with `currentColor`

**Comment by apinter** [comment]
> We should discuss adding a container section (#8) on the main page, and also container links/commands that you can easily copy on the pages of distributions (#4). Maybe we could also sync up on #3, since that could use a similar UI. I'm not that good at those things, so I would appreciate every bit of help

Happy to help with the technical bits, can't do much of anything about the marketing/filler text. I'm really bad at that.

File: _includes/icons/selfinstalling.svg (hunk 1 above)
Comment (apinter) [reply _includes/icons/selfinstalling.svg:1]: Cool, thanks, will update shortyl.

File: _includes/icons/selfinstalling.svg [hunk 2]
```diff
@@ -1 +1 @@
...
+<svg fill="currentColor" version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><path fill="none" d="M0 0h24v24H0z"/><path d="M9 2v2H5l-.001 10h14L19 4h-4V2h5a1 1 0 0 1 1 1v18a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1V3a1 1 0 0 1 1-1h5zm9.999 14h-14L5 20h14l-.001-4zM17 17v2h-2v-2h2zM13 2v5h3l-4 4-4-4h3V2h2z"/></svg>
```
Comment (hellcp) [inline _includes/icons/selfinstalling.svg:1]:
```suggestion
<svg fill="currentColor" version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><path d="M9 2v2H5l-.001 10h14L19 4h-4V2h5a1 1 0 0 1 1 1v18a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1V3a1 1 0 0 1 1-1h5zm9.999 14h-14L5 20h14l-.001-4zM17 17v2h-2v-2h2zM13 2v5h3l-4 4-4-4h3V2h2z"/></svg>
```
That's not needed

Comment (apinter) [reply _includes/icons/selfinstalling.svg:1]: I ended up symlinking the two, otherwise jekyll would error out when it can't find one or the other

Comment (apinter) [reply _includes/icons/selfinstalling.svg:1]: Was this change ok with you?

Comment (hellcp) [reply _includes/icons/selfinstalling.svg:1]: Yes, that being said, it doesn't resolve the above review

**Review by hellcp — APPROVED** [review]

**Pull request merged**